### DISCUSSION TITLE
Is that possible to run hypervisor on bottle rocket ?

### DISCUSSION THREAD

**Discussion by arul1712** [bottlerocket-os/bottlerocket#1829] [Q&A] [answered]
Please clarify on this

**Comment by zmrow** [comment, marked as answer]
I'll assume you're talking about type 1 hypervisors like KVM, Xen, etc.  Currently this is not possible with Bottlerocket, but is an interesting idea.  We currently target container workloads rather than virtualized workloads.

We would be love to hear more about your use case though!

**Reply by zmrow** [reply]
Currently the only container runtime we support is `containerd` (with the ECS variant also using Docker).

We'd also like to hear about your LXC use case if you're willing to provide additional details! :)
